centreon/www/include/configuration/configObject/hostgroup/DB-Func.php

```diff
@@ -152,30 +152,108 @@ function removeRelationLastHostgroupDependency(int $hgId): void
     }
 }
 
-function deleteHostGroupInDB($hostGroups = array())
+function deleteHostGroupInDB(array $hostGroups = [], bool $isCloudPlatform)
```

Comment:
Any optional arguments should be specified after any required arguments, otherwise they cannot be omitted from calls

Reply:
Done